lib/frontier_data_renderer/view_helper.rb

```diff
@@ -34,14 +34,10 @@ def format_value(value, type, opts)
   end
 
   def no_data_message(type)
-    content_tag(:span, no_data_text(type), class: "text-muted")
-  end
-
-  def no_data_text(type)
     if type == :boolean
-      "No"
+      content_tag(:span, "No", class: FrontierDataRenderer.no_data_class)
     else
-      "N/A"
+      content_tag(:abbr, "N/A", class: FrontierDataRenderer.no_data_class, title: "Not available")
```

Comment:
Fair enough :| That would mean the problem is further up the chain because the helper does not know if the data "not applicable" or "not available" either? Suggestions?